pkg/tfgen/docs_test.go

```diff
@@ -672,6 +672,43 @@ func TestParseAttributesReferenceSection(t *testing.T) {
 	assert.Len(t, ret.Attributes, 4)
 }
 
+func TestParseAttributesReferenceSectionParsesNested(t *testing.T) {
+	ret := entityDocs{
+		Arguments:  make(map[docsPath]*argumentDocs),
+		Attributes: make(map[string]string),
+	}
+	parseAttributesReferenceSection([]string{
+		"The following attributes are exported:",
+		"",
+		"* `id` - The ID of the Droplet",
+		"* `urn` - The uniform resource name of the Droplet",
+		"* `name`- The name of the Droplet",
+		"* `region` - The region of the Droplet",
+		"* `region.zone` - The zone of the Droplet region",
```

Comment:
Can we add a test for when `region.zone` comes before `region`?

Reply:
Added!